astro/src/content/blog/powerful-medical.mdx

```diff
@@ -0,0 +1,146 @@
+
+The challenge was compounded by modern engineering realities. Powerful Medical's team was migrating to containerized, cloud-native architectures while maintaining the flexibility to deploy on-premise for compliance-heavy healthcare customers. They needed authentication that could work seamlessly across development, testing, and production environments while integrating into their CI/CD pipelines.
+
+![FusionAuth's seamless developer experience](/img/blogs/powerful-medical/devex.png)
+
+## Why AWS Cognito Couldn't Scale to Enterprise Healthcare
+
+Powerful Medical had been using AWS Cognito for their individual user offering, but when they began developing their Enterprise platform for healthcare institutions, they found its fundamental limitations:
+
+**The Deal-Breaker**: No on-premise deployment capability. Many healthcare institutions require on-premise or air-gapped deployments for regulatory compliance. AWS Cognito simply couldn't meet this non-negotiable requirement.
+
+**Limited Feature Coverage**: Healthcare providers are notoriously process-heavy organizations with unique internal guidelines for everything, including authentication. Cognito does not support a large amount of authentication logic out of the box and often relies on workarounds like making customers self-develop the logic in Lambda functions. This would have been a lot of development overhead just to get common protocols working - protocols FusionAuth supports out of the box.
+
+**Integration Challenges**: Powerful Medical needed authentication that could integrate directly into their development and testing workflows—not just production.
+
+## The Search for Auth That's Theirs, Not Rented
+
+Faced with these constraints, Rovder embarked on an exhaustive evaluation of authentication providers. But this wasn't a typical vendor selection process—it was a comprehensive technical validation to ensure the chosen solution could meet every requirement of their demanding healthcare customers.
+
+> "We investigated basically all of the established IdP providers to see if they support all the features we anticipated we would need, and we actually tested out the features on all of them to see whether they live up to their documentation.”
+
+Their requirements list reflected the complex needs of both modern engineering teams and healthcare enterprises:
+
+**Core Technical Requirements:**
+
+- SAML and OIDC support
+- SCIM (syncing users and groups, plus deprovisioning)
+- Multi-factor authentication
+- Multiple authentication methods (username/password, email/magic link)
+- Customizable password requirements
+- Multi-tenant architecture with tenant-specific authentication rules
+- Source Code Escrow for enterprise compliance
+
+**The Non-Negotiable**: On-premise deployability
+
+**The Unique Need**: Support for the same user across multiple tenants (critical for their specific healthcare workflows)
```

Comment:
How did we handle this? It isn't regularly supported though there are workarounds.